Specs/Scene/Model/Model3DTileContentSpec.js

```diff
@@ -535,6 +537,15 @@ describe(
         });
       });
 
+      it("renders without normals", function () {
```

Comment:
This test fails even if I remove the `#ifdef HAS_NORMALS`.

Reply:
This test is using an i3dm tileset, so it's using `LegacyInstancingStageVS.glsl`, not `InstancingStageVS.glsl`.

If you were to add a line to the legacy version to include `attributes.normalMC`, then it would fail